packages/cf-component-card/src/CardDrawers.js

```diff
@@ -9,23 +9,40 @@ let UNIQUE_ID = 0;
 class CardDrawers extends React.Component {
   constructor(props) {
     super(props);
+
     this._cardId = UNIQUE_ID++;
+
+    this.state = {
+      active: null
+    };
+  }
+
+  handleLinkClick(id) {
+    this.props.onClick && this.props.onClick();
+
+    this.setState(state => ({
+      active: state.active === id ? null : id
+    }));
   }
 
   render() {
+    const active = this.props.hasOwnProperty('active')
```

Comment:
When you pass `undefined` I would expect that

```js
active = this.state.active;
```

so it behaves the same as in a case the prop `active` was left out completely.

When `null`

```js
active = this.props.active;
```

But here it will assign `this.props.active` in both cases. `undefined` means to me the prop doesn't exist and the component should use its own state. `null` means the prop exist but we want no drawer to be open.

Your call. If this works for you I'm fine.